libmscore/score.cpp

```diff
@@ -1469,7 +1469,7 @@ void Score::removeElement(Element* element)
             measures()->remove(mb);
             System* system = mb->system();
             Page* page = system->page();
-            if (element->isVBox() && system->measures().size() == 1) {
+            if (element->isVBox() || element->isTBox() && system->measures().size() == 1) {
```

Comment:
Yes, but you are missing parentheses around `element->isVBox() || element->isTBox()`.

Reply:
:facepalm: sorry, will fix